nixos/modules/services/networking/cjdns.nix

```diff
@@ -245,7 +245,7 @@ in
         fi
 
         if [ -z "$CJDNS_ADMIN_PASSWORD" ]; then
-            echo "CJDNS_ADMIN_PASSWORD=$(tr -dc A-Za-z0-9 </dev/urandom | head -c 96)" \
+            echo "CJDNS_ADMIN_PASSWORD=$(tr -dc A-Za-z0-9 </dev/urandom | head -c 32)" \
```

Comment:
Didn't you want to limit to 64?

Reply:
Ah didn't change PR, changed commit only. After checking found out that it was not working with 64 and 63, so went down to 32.